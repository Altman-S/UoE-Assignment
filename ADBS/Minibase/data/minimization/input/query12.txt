Q(x, z) :- R(4, z), S(x, y), T(12, 'x'), R(u, z), R(u, z), S(x, 4), T(14, 'y')