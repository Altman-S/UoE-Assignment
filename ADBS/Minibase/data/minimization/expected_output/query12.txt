Q(x, z) :- R(4, z), T(12, 'x'), S(x, 4), T(14, 'y')